### PR TITLE
Responsive NavBar

### PR DESCRIPTION
**Pull request by jimmyygit**
Fixes issues with horizontal scrolling and navbar. 
Some excess space was caused by large title fonts on smaller viewports; media query to adjust.
 There are still some spacing issues caused by the Contact Form; will look into future pr

### CODE REVIEW

**Comment by functionstackx** [comment]
@jimmyygit is there a way to make the phone mockup appear on mobile too or do you think it is better without it on mobile?

image

**Comment by functionstackx** [comment]
@jimmyygit also is there a way to make the whitespace a bit more reasonable?

image

**Comment by jimmyygit** [comment]
> @jimmyygit is there a way to make the phone mockup appear on mobile too or do you think it is better without it on mobile?
> 
> image

@OrenLeung I can put it in if we want. It was a design choice for me to hide the phone mockup on mobile because it would get very chaotic with the header text.

**Comment by functionstackx** [comment]
> @OrenLeung I can put it in if we want. It was a design choice for me to hide the phone mockup on mobile because it would get very chaotic with the header text.

i think it is fine as is, just wondering.

**Comment by jimmyygit** [comment]
> @jimmyygit also is there a way to make the whitespace a bit more reasonable?
> 
> image

oh lol. I'll adjust the spacing. shouldnt be hard

File: components/Contact.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ const Contact = () => {
               />
             </Form.Group>
           </Form.Row>
-          <HomeMadeButton type="purpleWhite" link="/">
+          {/* <HomeMadeButton type="purpleWhite" link="/">
```
Comment (LemonFace0309) [inline components/Contact.js]: are you planning to remove the comments later or are you gonna keep it there for now?

File: components/shared/TextAndPic.module.css [hunk 2]
```diff
@@ -21,7 +21,7 @@
 @media (max-width: 768px) {
   .ImageWrapper {
     margin: 0;
-    height: 60vh;
+    /* height: 60vh; */
```
Comment (LemonFace0309) [inline components/shared/TextAndPic.module.css:24]: same comment thing here

**Review by LemonFace0309 — CHANGES_REQUESTED** [review]

**Review by LemonFace0309 — APPROVED** [review]

File: components/Contact.js (hunk 1 above)
Comment (jimmyygit) [reply components/Contact.js]: I think I was just testing and forgot to comment back in

File: components/shared/TextAndPic.module.css (hunk 2 above)
Comment (jimmyygit) [reply components/shared/TextAndPic.module.css:24]: this one i will delete

**Pull request merged**